config.test.analytics.yaml

```diff
@@ -0,0 +1,124 @@
+# RESTBase config for Analytics APIs
```

Comment:
There's no need to define a separate `config.test.analytics.yaml`, as it's not picked up anywhere, therefor tests are failing in travis. Just adding 

```yaml
    /{module:pageviews}:
        x-modules:
            - name: pageviews
              version: 1.0.0
              type: file
```
to the `config.test.yaml` would be enough.